packages/runtime/container-runtime/src/summary/summaryGenerator.ts

```diff
@@ -184,8 +185,7 @@ export class SummarizeResultBuilder {
 /**
  * Errors type for errors hit during summary that may be retriable.
  */
-export class RetriableSummaryError extends LoggingError {
-	public readonly canRetry = this.retryAfterSeconds !== undefined;
+export class RetriableSummaryError extends LoggingError implements IRetriableSummaryError {
```

Comment:
There is nothing summary related in this error. We should either have ay summary related information in this error or should rename the class.

Reply:
Like renaming it to IFluidRetriableError or adding an errorType in the interface.

Reply:
Adding an errorType like "summaryRetriableError" should be good imo.

Reply:
For the errorType, there is a discussion going on here -
https://github.com/microsoft/FluidFramework/pull/20851/files#r1598963838. I could add one but I don't see a lot of value in making it part of `alpha` API because that means we have to support it forever.

Reply:
And the reason I added `Summary` to the name is because it is only used for summary failures. I could name it more generically, I don't have any strong preference.